health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/MyBillsTable.js

```
@@ -128,11 +128,12 @@ const MyBillsTable = ({ ...props }) => {
                         reportDetails?.status === "COMPLETED" ? <Button
                             className="custom-class"
                             iconFill=""
+                            size="medium"
                             icon="FileDownload"
                             isSuffix
                             label={t(`HCM_AM_DOWNLOAD_BILLS`)}
                             title={t(`HCM_AM_DOWNLOAD_BILLS`)}
-                            showBottom={isLastRow ? false : true}
+                            showBottom={isLastRow && props.data.length !== 1 ? false : true}
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)